File: Aula_04/python/02.py
class Viagem:
    def __init__ (self):
        self.distancia = 0
        self.tempo_horas = 0
        self.tempo_minutos = self.tempo_horas * 60
    def calc_velocidade (self):
        return self.distancia / self.tempo_horas

v = Viagem()
v.distancia = float(input("Digite a distancia da viagem: "))
v.tempo_horas = float(input("Digite o tempo da viagem em horas: "))
print(f"Velocidade media da viagem = {v.calc_velocidade()}")
print(f"Tempo da viagem em minutos = {v.tempo_minutos}")

File: Aula_04/c++/02.cpp
#include <iostream>

class Viagem{
    public:
    double distancia = 0;
    int tempo_hora = 0, tempo_minuto = 0;

    double calc_velocidade(){
        double tempo_total = tempo_hora + (tempo_minuto / 60.0);
        return distancia / tempo_total;
    }
};

int main(){
    Viagem v;

    std::cout << "Digite a distancia de uma viagem de km: ";
    std::cin >> v.distancia;

    std::cout << "Digite o numero de horas demora a viagem: ";
    std::cin >> v.tempo_hora;

    std::cout << "Digite o numero de minutos demora a viagem: ";
    std::cin >> v.tempo_minuto;

    std::cout << "Velocidade media da viagem = " << v.calc_velocidade() << " km/h" << std::endl;
    return 0;
}